libs/dyn/jsonloader/json.go

```diff
@@ -70,7 +70,9 @@ func decodeValue(decoder *json.Decoder, o *Offset) (dyn.Value, error) {
 					return invalidValueWithLocation(decoder, o), err
 				}
 
-				obj.Set(keyVal, val)
+				if err := obj.Set(keyVal, val); err != nil {
+					return invalidValueWithLocation(decoder, o), err
```

Comment:
we need to just log error here and not return an error, otherwise it breaks the decoding of all the value